lib/store.js

```diff
@@ -83,6 +83,14 @@ Store.prototype.del = function (packet, cb) {
 }
 
 /**
+ * deletes a packet from the store by messageId
+ */
+Store.prototype.delByMessageId = function (messageId) {
+  delete this._inflights[messageId]
+  return this
+}
```

Comment:
This new method is not needed, you can use https://github.com/redboltz/MQTT.js/blob/c35e0a615b44bb235fecffb6b2ae8bdee4556c5a/lib/store.js#L74 and call it with `{ messageId }`.